File: Basic-Programs/Program25.c
#include<stdio.h>

void DisplayResult(int iStandard)
{
    if(iStandard <= 0)
    {
        iStandard = - iStandard;        // Updator 
    }
    
    switch(iStandard)
    {
        case 8:
        case 9:
            printf("Your Exam At 9am \n");
            break;
            
        case 10:
        case 11:
            printf("Your Exam At 11am \n");    
            break;
            
        case 12:
            printf("Your Exam At 1pm \n");    
            break;
            
        default:
            printf("Sorry! We don't have information about your Standard \n");   
    }
}

int main()
{
    int iValue = 0;
    
    printf("Enter Your Standard : \n");
    scanf("%d",&iValue);
    
    DisplayResult(iValue);
    
    return 0;
}

File: Basic-Programs/Program25.java

import java.util.Scanner;

class Demo
{
    public int iNo = 0;
    
    Demo(int iValue)
    {
        iNo = iValue;
    }
    
    public void DisplayResult()
    {
        if(iNo <= 0)
        {
            iNo = - iNo;        // Updator 
        }
        
        switch(iNo)
        {
            case 8:
            case 9:
                System.out.println("Your Exam At 9am");
                break;
                
            case 10:
            case 11:
                System.out.println("Your Exam At 11am");
                break;
                
             case 12:
                 System.out.println("Your Exam At 1pm");
                 break;
                 
              default:
                  System.out.println("Sorry! we don't have information about your Standard");
                  break;   
        }
    }
    
}

class Program25
{
    public static void main(String A[])
    {
        int iStandard = 0;
        
        Scanner Sobj = new Scanner(System.in);
        System.out.println("Enter Your Standard : ");
        iStandard = Sobj.nextInt();
        
        Demo dobj = new Demo(iStandard);
        dobj.DisplayResult();
                
    }
}